lib/onix/product.rb

```diff
@@ -809,5 +799,81 @@ def parse(n)
       end
 
     end
+
+    def manage_prices(supplies, global_price)
```

Comment:
Un petit commentaire du genre "add missing from_date and until_date when they can be guessed" ?